core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1246,6 +1246,14 @@ package object config {
       "mechanisms to guarantee data won't be corrupted during broadcast")
     .booleanConf.createWithDefault(true)
 
+  private[spark] val BROADCAST_FOR_UDF_COMPRESSION_THRESHOLD =
+    ConfigBuilder("spark.broadcast.UDFCompressionThreshold")
+      .doc("The threshold at which a a user-defined function (UDF) is compressed by broadcast, " +
```

Comment:
I think this also applies to RDD APIs. We can just say, for instance, `The threshold at which Python commands for RDD APIs and user-defined function (UDF) are serialized by broadcast ...`. Feel free to change wording.

Reply:
I updated the description to include this.